packages/ia-components/sandbox/audio-players/youtube-player.jsx

```diff
@@ -92,23 +99,25 @@ class YoutubePlayer extends Component {
    */
   loadPlayer() {
     const { id, playerAnchor } = this.state;
-    this.setState({
-      player: new window.YT.Player(playerAnchor.current, {
-        height: '600',
-        width: '600',
-        videoId: id,
-        playerVars: {
-          fs: 1,
-          rel: 0,
-          enablejsapi: 1
-        },
-        events: {
-          onReady: this.onPlayerReady,
-          onStateChange: this.onPlayerStateChange,
-          onError: this.onPlayerError
-        }
-      })
+    const { id: propsID } = this.props;
+    const idToUse = id || propsID;
```

Comment:
**Suggestion**: rename to videoId to take advantage of property definition shorthand.